services/madoc-ts/src/frontend/shared/hooks/use-data.ts

```diff
@@ -186,7 +186,7 @@ export function useInfiniteData<Data = any, TKey = any, TVariables = any>(
       }
       return undefined as any;
     },
-    { ...(config || {}), useErrorBoundary: true }
+    { ...(config || {}), useErrorBoundary: true, keepPreviousData: true }
```

Comment:
I think this might cause stale data issues in other places, you should be able to pass it into individual hooks if required.